next/CMakeLists.txt

```diff
@@ -0,0 +1,137 @@
+
+write_basic_package_version_file(
+    "${CMAKE_CURRENT_BINARY_DIR}/hipblas-common-config-version.cmake"
+    VERSION ${PROJECT_VERSION}
+    COMPATIBILITY ExactVersion
+  )
+install(
+    TARGETS hipblas-common
+    EXPORT hipblas-common-targets
+    COMPONENT devel
+)
+install(
+    EXPORT hipblas-common-targets
+    DESTINATION "${CMAKE_INSTALL_LIBDIR}/cmake/hipblas-common"
+    NAMESPACE roc::
+    COMPONENT hipBLAS-common_Development
+    FILE hipblas-common-config-targets.cmake
+)
+install(FILES
+    "${CMAKE_CURRENT_BINARY_DIR}/hipblas-common-config-version.cmake"
+    "${CMAKE_CURRENT_SOURCE_DIR}/../cmake/hipblas-common-config.cmake"
+    DESTINATION ${CMAKE_INSTALL_LIBDIR}/cmake/hipblas-common)
+install(
+    FILES ${hipblas_headers_public}
+    COMPONENT hipBLAS-common_Development
+    DESTINATION "${CMAKE_INSTALL_INCLUDEDIR}/hipblas-common"
+)
+install(
+    FILES "${CMAKE_CURRENT_SOURCE_DIR}/../LICENSE.md"
+    COMPONENT hipBLAS-common_Docs
+    DESTINATION "${CMAKE_INSTALL_DOCDIR}"
+)
+export(EXPORT hipblas-common-targets FILE hipblas-common-config.cmake)
+
+set(CPACK_PACKAGE_NAME hipBLAS-common)
+set(CPACK_PACKAGE_VENDOR "Advanced Micro Devices, Inc")
+set(CPACK_PACKAGE_CONTACT "hipBLAS Maintainer <hipblas-maintainer@amd.com>")
+set(CPACK_PACKAGE_DESCRIPTION_SUMMARY "Common files shared by hipBLAS and hipBLASLt")
+set(CPACK_PACKAGE_INSTALL_DIRECTORY ${CPACK_PACKAGE_NAME})
+set(CPACK_PACKAGE_VERSION_MAJOR ${PROJECT_VERSION_MAJOR})
+set(CPACK_PACKAGE_VERSION_MINOR ${PROJECT_VERSION_MINOR})
+set(CPACK_PACKAGE_VERSION_PATCH ${PROJECT_VERSION_PATCH})
+set(CPACK_VERBATIM_VARIABLES YES)
+set(CPACK_RESOURCE_FILE_LICENSE ${CMAKE_CURRENT_LIST_DIR}/../LICENSE.md)
+set(CPACK_RESOURCE_FILE_README ${CMAKE_CURRENT_LIST_DIR}/../README.md)
+set(CPACK_COMPONENTS_ALL "hipBLAS-common_Development;hipBLAS-common_Docs")
+set(CPACK_GENERATOR "TGZ;ZIP;RPM;DEB")
+
+set(CPACK_DEBIAN_COMPRESSION_TYPE "xz" CACHE STRING "") # is this important? Should it be configurable? Should we mark as advance? Came from rocm-cmake...
+set(CPACK_DEBIAN_FILE_NAME "DEB-DEFAULT")
+set(CPACK_DEBIAN_PACKAGE_PROVIDES "hipblas-common (= 1.0.0)")
+set(CPACK_DEB_COMPONENT_INSTALL "ON")
+
+set(CPACK_DEBIAN_HIPBLAS-COMMON_DEVELOPMENT_PACKAGE_NAME "hipblas-common-dev")
+set(CPACK_DEBIAN_HIPBLAS-COMMON_DOCS_PACKAGE_NAME "hipblas-common-doc")
+set(CPACK_DEBIAN_HIPBLAS-COMMON_DOCS_PACKAGE_SECTION "doc")
+#set(CPACK_DEBIAN_PACKAGE_RELEASE "some git hash") # there is come etra functionality in rocm-cmake to compute this hash why not use the command line?
+#set(CPACK_DEBIAN_PACKAGE_DEPENDS "rocm-core") # I don't think we really need this to build
+
+set(CPACK_RPM_COMPONENT_INSTALL "ON")
+set(CPACK_RPM_FILE_NAME "RPM-DEFAULT")
+set(CPACK_RPM_EXCLUDE_FROM_AUTO_FILELIST_ADDITION "${CPACK_PACKAGING_INSTALL_PREFIX}")
+set(CPACK_RPM_PACKAGE_AUTOREQPROV "Off")
+set(CPACK_RPM_PACKAGE_LICENSE "MIT")
+#set(CPACK_RPM_PACKAGE_RELOCATABLE "Off") why?
```

Comment:
If we don't think we need things, let's leave them commented out with a note that they differ from the prior version. If it passes QA/testing and no one remembers why they were there, then we don't add them back.